concepts/string.md

```diff
@@ -46,13 +48,16 @@ ASCII only has 128 characters.
 - the letters of the Latin alphabet required for basic English
 - common punctuation characters
 - control characters
+
 ASCII cannot represent `β`, `喂`, or even `é`.
 
-In the early days of the internet, multiple standards evolved to represent text content in more languages.
-In the 2000s, the **UTF-8** standard became the most popular encoding on the internet.
+As the computer world became more international, a better character set quickly became necessary. Early attempts consisted in extending ASCII to multiple code pages, with each codepage being able to represent a group of characters used in a given set of languages.  That started with DOS, and continued with Windows 3.x and future versions. But that was poorly interchangeable, as the same byte values could represent different characters depending on which codepage was used.
+
+In 1991, a new standard emerged called Unicode.  Its goal was to represent as many characters as possible in a single character set and solve the interoperability issue. Its first version contained over 7000 characters.  As of September 2022, Unicode is in version 15.0 and contains 149,186 characters or codepoints!
+
+Unicode has many different encodings. The most common one is **UTF-8**, but other exist like **UTF-16** or even **UTF-32**.  UTF-8 is a variable length encoding, while UTF-32 is fixed length (as long as the character set doesnt try to represent more than 4 billion chars!). UTF-32 is a memory hog, but it is predictable since it's fixed length and it has the advantage that the byte values are an exact match to the codepoint value.  UTF-16 is also variable length, not fixed length, a mistake often made by newbie developers.  In UTF-16, in order to represent characters beyond the first plane of Unicode (Basic Multilingual plane), the encoding uses **surrogate pairs**.  Don't ever assume that a character is always encoded with 2 bytes in UTF-16!
```

Comment:
This information is super interesting, but I fear it might not be a little out of scope for Machine Translate, as this article is about strings.
Please bear in mind that we try to [keep content minimal](https://machinetranslate.org/style#machine-translation-ready).
Do you think we can summarize this part into fewer sentences, just to give minimal history?